vendor/assets/stylesheets/dvl/components/page_header.scss

```diff
@@ -33,35 +36,92 @@
   }
 
   > .button,
-  > form .button {
+  > form .button,
+  > .button_uppercase {
     float: left;
     clear: left;
-    margin: $rhythm 0;
     @media screen and (min-width: $lapWidth) {
       float: right;
       clear: none;
       position: relative;
-      margin: 0;
-      top: ($fontLineHeightH2 - $inputHeight) / 2;
+      @include topCenter($fontLineHeightH2, $inputHeight);
       &.small {
-        top: ($fontLineHeightH2 -($rhythm * 4)) / 2;
+        @include topCenter($fontLineHeightH2, ($rhythm * 4));
       }
       &.mini {
-         top: ($fontLineHeightH2 - ($rhythm * 3)) / 2;
+         @include topCenter($fontLineHeightH2, ($rhythm * 3));
       }
     }
   }
 
+  > .button,
+  > form .button {
+    margin: $rhythm 0;
+    @media screen and (min-width: $lapWidth) {
+      margin: 0;
+    }
+  }
+
+  > .button_uppercase {
+    margin-bottom: $rhythm;
+    @media screen and (min-width: $lapWidth) {
+      margin: ($rhythm * 1.5) 0;
+    }
+  }
+
   // Modifiers
 
   &.with_back_arrow {
+    margin-top: $rhythm;
```

Comment:
@jrubenoff: Can you explain why this is here? This leads to some unwanted behavior in Screendoor:

When changing from the "New project" to "Example projects" page, the page header jumps down, even though there's no context being added on the example projects page.

Screencast: http://take.ms/ry8LB